eng/pipelines/templates/steps/sync-directory.yml

```diff
@@ -3,14 +3,15 @@ parameters:
   DirectoryToSync: eng/common
   CommitMessage: commit-message-not-set
   UpstreamBranchName: branch-name-not-set
-  BaseBranchName: master
+  BaseBranchName: $(DefaultBranch)
   Sync: pr-data-artifact-path-not-set
   SkipCheckingForChanges: false
   ScriptDirectory: eng/common/scripts
   PROwner: Azure
   PushArgs: -f
 
 steps:
+- template: /eng/common/pipelines/templates/steps/set-default-branch.yml
```

Comment:
I think this needs to be pulled into the inner loop.

Reply:
Put it after clone step